app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -228,6 +231,8 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
      */
     private int minimumHeightOfMetadata = 200;
 
+    final static String NOMINATING_MEDIA = "Nominating %s";
```

Comment:
Would you mind making `NOMINATING_FOR_DELETION_MEDIA = "Nominating for deletion %s"`? It will be more understandable. Thanks!